easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb

```diff
@@ -29,7 +28,7 @@ buildopts = ' STAR && make STARlong'
 parallel = 1
 
 files_to_copy = [(["source/STAR", "source/STARlong"], "bin"), "CHANGES.md", "doc", "extras", "LICENSE", "README.md", "RELEASEnotes.md"]
```

Comment:
@wpoely86 The style tests are not enabled yet, we need to resolve over 1k style issues in the current easyconfigs collection before we can do that...

I'm gradually working towards making `--style-check` pass on all the easyconfigs we have now, with the intention to enable the style checks ASAP